lms/services/course.py

```diff
@@ -156,6 +170,23 @@ def _is_pre_sections(self, context_id):
             )
         )
 
+    def _get_copied_from_course(self, lti_params) -> Course | None:
```

Comment:
This method is very similar to the one for assignments:


https://github.com/hypothesis/lms/blob/course-copied-from/lms/services/assignment.py#L65